feature/experimental/policy/policy_vrf_selection/ate_tests/base_vrf_selection/base_vrf_selection_test.go

```diff
@@ -335,13 +367,13 @@ func contains(item string, items []string) bool {
 	return flag
 }
 
-// verifyTraffic confirms that every traffic flow has the expected amount of loss (0% or 100%
-func verifyTraffic(t *testing.T, ate *ondatra.ATEDevice, flows []*ondatra.Flow, passFlows []string) {
+// verifyTraffic confirms that every traffic flow has the expected amount of loss (0% or 100%).
+func verifyTraffic(t *testing.T, ate *ondatra.ATEDevice, flows []*ondatra.Flow, passFlows []*ondatra.Flow) {
```

Comment:
don't need to repeat the type on flows and passFlows, Just "flows, passFlows []*ondatra.Flow" is fine

Reply:
done. in new commit.,